.env.example

```
@@ -72,6 +72,8 @@ REDIS_URL=redis://localhost:6379
 VITE_PUBLIC_API_URL=http://localhost:3001
 VITE_PUBLIC_WEB_URL=http://localhost:3000
 VITE_PUBLIC_APP_NAME=twotter
+# Databuddy analytics client ID (from https://app.databuddy.cc). Leave blank to disable.
+VITE_PUBLIC_DATABUDDY_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_PUBLIC_DATABUDDY_CLIENT_ID key should go before the VITE_PUBLIC_WEB_URL key

(UnorderedKey)